gef.py

```diff
@@ -2068,8 +2068,11 @@ def gdb_get_nth_previous_instruction_address(addr: int, n: int) -> Optional[int]
 def gdb_get_nth_next_instruction_address(addr: int, n: int) -> int:
     """Return the address (Integer) of the `n`-th instruction after `addr`."""
     # fixed-length ABI
+    if n == 0:
+        raise ValueError(f"`n` must be strictly positive")
+
     if gef.arch.instruction_length:
-        return addr + n * gef.arch.instruction_length
+        return addr + (n-1) * gef.arch.instruction_length
```

Comment:
So if n == 1 we just return addr?

that is OK, but aren't we changing the behaviour, where we were basically zero-indexed before? Looks like it's only used by nop and skipi, so I think this is good. Just a bit surprised it ever worked, then?

Reply:
AAMOF I'll update: from the docstrings, the function was meant to be zero-indexed.